python/nav/web/api/v1/urls.py

```diff
@@ -73,4 +73,5 @@
         name="prefix-usage-detail",
     ),
     re_path(r'^', include(router.urls)),
+    re_path(r'^vendor/?$', views.VendorLookup.as_view(), name='vendor'),
```

Comment:
My DRF-fu is rusty (as mentioned in another PR review today).  I assume the primary reason the url pattern is added explicitly, rather than through DRF's router-thingy is that this view isn't actually a model view with built-in CRUD-magic?

Reply:
yes, using the router stuff did not seem to work very well with `APIView`